modules/ROOT/pages/app-to-app-requests.adoc

```diff
@@ -1,4 +1,4 @@
-= Access One Application from Another in the Same Cluster
+= Sending Application-to-Application Request in the Same Cluster
```

Comment:
Sending **an** Application-to-Application Request in the Same Cluster or Sending Application-to-Application Request**s** in the Same Cluster (?)